subsys/bluetooth/controller/Kconfig

```diff
@@ -545,7 +551,7 @@ config BT_CTLR_SYNC_PERIODIC
 	  Synchronization state in the Controller.
 
 config BT_CTLR_SYNC_PERIODIC
-	bool "LE Periodic Advertising in Synchronization State [EXPERIMENTAL]" if BT_LL_SW_SPLIT
+	bool "LE Periodic Advertising in Synchronization State (Split Link Layer) [EXPERIMENTAL]" if BT_LL_SW_SPLIT
```

Comment:
same here. not related to this line, but below.

You still need:
https://github.com/zephyrproject-rtos/zephyr/blob/65a659e7f7e2ad76a2406f8d6db57bca99ee25b3/subsys/bluetooth/controller/Kconfig#L563-L564

just without the help text.

Notice the two config BT_CTLR_SYNC_PERIODIC after one another, and then a depends on BT_CTLR_SYNC_PERIODIC here:
https://github.com/zephyrproject-rtos/zephyr/blob/e4e8b78f93a7871c4494bf6b2e5596e97fcf7c12/subsys/bluetooth/controller/Kconfig#L544-L559